src/routes/api/api-router.ts

```diff
@@ -91,6 +91,13 @@ ApiRouter.delete(
 	returnOnValidationError,
 	ApiInstallationDelete
 );
+ApiRouter.delete(
+	"/deleteInstallation/:installationId/:jiraHost/github-app-id/:gitHubAppId",
+	param("installationId").isInt(),
+	param("jiraHost").isString(),
```

Comment:
comment on pre-existing code not this one lollol

since the jirahost includes the protocol do we assume it's been encoded? the delete method appears to just take it as is? I feel like i missed a piece but putting it here just in case.

Reply:
Yup good point 👍 
I think "it is working", if you check the test added in the PR, we are assuming that the `jiraHost` is encoded. And also assuming express decoded it automatically.

ps: and since it is pre-exsit code, working fine with pollinator, so we have a bit confidence.

Reply:
yeah agree